splink/vertically_concatenate.py

```diff
@@ -14,7 +15,11 @@
     from .linker import Linker
 
 
-def vertically_concatenate_sql(linker: Linker) -> str:
+def vertically_concatenate_sql(
```

Comment:
Main changes are here - rather than accepting a linker, this now accepts arguments, meaning it can be used outside the context of a linker